compiler+runtime/test/bash/module/clojure-string/src/clojure_string/self_test.jank

```diff
@@ -0,0 +1,55 @@
+(ns clojure-string.self-test
```

Comment:
Please put these tests in the https://github.com/jank-lang/clojure-test repo instead. We should have one place for testing all of the official Clojure namespaces.

Reply:
https://github.com/jank-lang/clojure-test/pull/70

Reply:
Would you accept these tests with this disclaimer? Every test namespace fails use to #226. 

(Line 4)

```
;;TODO delete this namespace and use clojure-test-suite
;;https://github.com/jank-lang/jank/issues/226
```

Reply:
That is a critical bug which will be fixed shortly. It's already assigned to someone. We don't benefit from having these tests between now and then. I would rather we not duplicate tests from clojure--test-suite at all.

We'll get there, so that we can run the whole test suite. We don't want to accrue a bunch of duplication along the way.

Reply:
Sounds good, removed.